src/lib/serviceRoutines/putIndividualContextEntity.cpp

```diff
@@ -39,6 +40,22 @@
 /* ****************************************************************************
 *
 * putIndividualContextEntity - 
+*
+* Corresponding Standard Operation: UpdateContext/UPDATE
+*
+* PUT /v1/contextEntities/{entityId::id}
+* PUT /ngsi10/contextEntities/{entityId::id}
+*
+* Payload In:  UpdateContextElementRequest
+* Payload Out: UpdateContextElementResponse
+*
+* URI parameters:
+*   - attributesFormat=object
```

Comment:
What about the following filters:

- !exist=entity::type
- exist=entity::type
- entity::type=TYPE

Note that to modify entities without type you need the !exist=entity::type in the PUT operation.

What about the same for POST? Not sure if POST logic has been modified yet (i.e. the PR for POST operation has been issued or not).